airqo-platform/AirQo-frontend
# Web update 27042021

#### Summary of Changes (What does this PR do?)
Updating the web repository to latest from online

#### Status of maturity (all need to be checked before merging):

- Tested it locally 
- Same as it is online

#### How should this be manually tested?
Fetch this branch locally.
* Set `.env` with respective values ([slack](https://airqo.slack.com/archives/GTHGHCB4G/p1618994366002700))
* Build app `docker-compose build`
* Run App `docker-compose up`
* * Check the website (http://localhost:8000/).
* Compare the local app with the actual website (airqo.net)

#### Screenshots (optional)
